lawnchair/src/app/lawnchair/ui/preferences/AppDrawerPreferences.kt

```diff
@@ -61,18 +55,35 @@ fun AppDrawerPreferences() {
             )
             SuggestionsPreference()
         }
+
```

Comment:
```suggestion

        val showDrawerSearchBar = !prefs2.hideAppDrawerSearchBar.getAdapter()
```

(You can choose where to put this though)